### DISCUSSION TITLE
i cannot install tailwindcss to laravel

### DISCUSSION THREAD

**Discussion by neondev27** [tailwindlabs/tailwindcss#8533] [Help] [answered]
**What version of Tailwind CSS are you using?**

3.0.24

**What build tool (or framework if it abstracts the build tool) are you using?**

Laravel 9.x

**What version of Node.js are you using?**

v12.14.1

**What browser are you using?**

Opera

**What operating system are you using?**

Windows 10

**Reproduction URL**

İ followed the steps until last step, but when i run "npm run watch", i get this error:

> @ watch C:\Users\billab01\Desktop\communidia.one(alpha release 0.0.0.1)\communidia-alpha-0.0.0.1
> mix watch

'mix' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ watch: `mix watch`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ watch script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\billab01\AppData\Roaming\npm-cache\_logs\2022-06-07T06_22_00_137Z-debug.log

could you help me?

**Comment by divyeshchotaliya** [comment, marked as answer]
**Make sure you have installed `laravel-mix` package.**

I got the same error while running `npm run watch` command and I got a solution after running `npm install`.

Because I forgot to run this command before installing tailwindcss in Laravel project. that's why they can't found `laravel-mix` package and throw an error.

laravel provide `laravel-mix` package in package.json file by default. so just run `npm install`

**Reply by neondev27** [reply]
thanks
